### PR TITLE
Align plug-in descriptions

### PR DESCRIPTION
**Pull request by infotexture**
This PR updates several plug-in descriptions to align with recent changes in the documentation.

### CODE REVIEW

**Comment by infotexture** [comment]
@jelovirt Just realized we might want to wire this repo up to Slack so that future PRs trigger notifications either in the `#development` channel there, or in a dedicated `#registry` channel.

File: com.sophos.tocjs.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 [
   {
     "name": "com.sophos.tocjs",
-    "description": "The tocjs plugin generates a Table of Contents for the DITA Open Toolkit's XHTML output that can expand and collapse nodes. It is a more usable and visually pleasing option than the standard index.html generated by the Open Toolkit's dita2html transformation.",
-    "keywords": ["XHTML"],
+    "description": "The TocJS plugin generates a JavaScript-based frameset for the DITA Open Toolkit's HTML5 output with Table of Contents entries that expand and collapse.",
+    "keywords": ["HTML5"],
```
Comment (jelovirt) [inline com.sophos.tocjs.json]: TocJS plugin does not output HTML5, it uses XHTML.

Comment (infotexture) [reply com.sophos.tocjs.json]: @jelovirt Turns out the **XHTML → HTML5** change came from you in [dita-ot/docs@54fb866](https://github.com/dita-ot/docs/commit/54fb866a5a3ac592051d30f74ad4e0c3643c0bf8#diff-6f6633201b886faa05ca1a21de12514a).

Can you confirm which is correct?

Comment (jelovirt) [reply com.sophos.tocjs.json]: My bad, it's still XHTML. https://github.com/dita-ot/com.sophos.tocjs/blob/develop/integrator.xml#L68 calls https://github.com/dita-ot/dita-ot/blob/develop/src/main/plugins/org.dita.xhtml/build_general_template.xml#L118

Comment (infotexture) [reply com.sophos.tocjs.json]: Restored XHTML references in e955eef.

**Pull request merged**